dotcom-rendering/src/web/components/ArticleHeadline.stories.tsx

```diff
@@ -313,27 +313,42 @@ export const Comment = () => {
 Comment.story = { name: 'Comment' };
 
 export const Analysis = () => {
-	const format = {
+	const themes: [string, ArticleTheme][] = [
+		['News', ArticlePillar.News],
+		['Opinion', ArticlePillar.Opinion],
+		['Sport', ArticlePillar.Sport],
+		['Culture', ArticlePillar.Culture],
+		['Lifestyle', ArticlePillar.Lifestyle],
+	];
+	const format = (theme: ArticleTheme): ArticleFormat => ({
 		display: ArticleDisplay.Standard,
 		design: ArticleDesign.Analysis,
-		theme: ArticlePillar.News,
-	};
+		theme,
+	});
+
 	return (
-		<ElementContainer>
-			<Flex>
-				<LeftColumn borderType="full">
-					<></>
-				</LeftColumn>
-				<ArticleContainer format={format}>
-					<ArticleHeadline
-						headlineString="This is an Analysis headline, it's underlined. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor"
-						palette={decidePalette(format)}
-						format={format}
-						tags={[]}
-					/>
-				</ArticleContainer>
-			</Flex>
-		</ElementContainer>
+		<>
+			{themes.map(([pillar, theme]) => (
```

Comment:
🤔 what is `pillar` here. Isn't `pillar` a subset of `theme`?

Reply:
Changed to `themeName`!

Reply:
Ah! 🤦 I see now